src/utils/network-monitor.ts

```diff
@@ -979,9 +984,17 @@ export class NetworkMonitor {
     if (this.blockJobs[network].length > 0) {
       const blockJob: BlockJob = this.blockJobs[network][0] as BlockJob
       if (this.enableV2) {
-        this.processBlock2(blockJob)
+        try {
+          await this.processBlock2(blockJob)
+        } catch (error: any) {
+          this.structuredLogError(blockJob.network, `Error processing block: ${JSON.stringify(error)}`, blockJob.block)
```

Comment:
Not sure this is necessary. I will attempt to make the error logs more consistent on next pass over